lib/bgsite_official/categories/tag.ex

```diff
@@ -23,7 +23,7 @@ defmodule BgsiteOfficial.Categories.Tag do
   @doc false
   def changeset(tag, attrs) do
     tag
-    |> cast(attrs, [:title, :banner, :description, :likes, :priority])
```

Comment:
IMO we have to leave `:priority` to be cast when entered. So just remove it from `required`.